sra_experiment/1_download_process_data.ipynb

```diff
@@ -0,0 +1,12880 @@
+      "-----------------------------------------\n",
+      "size = 5641451\n",
+      "Number of ones: 7451\n",
+      "Number of ones per inventory item: 512\n",
+      "Inventory entries filled: 15\n",
+      "-----------------------------------------\n",
+      "| Loading contig boundaries | Time = 8.0762 ms\n",
+      "-----------------------------------------\n",
+      "size = 5641451\n",
+      "-----------------------------------------\n",
+      "| Loading sequence | Time = 444.98 us\n",
+      "-----------------------------------------\n",
+      "size = 5417921\n",
+      "[2020-07-10 17:59:46.860] [jointLog] [info] Loading pufferfish index\n",
+      "[2020-07-10 17:59:46.860] [jointLog] [info] Loading dense pufferfish index.\n",
+      "-----------------------------------------\n",
+      "| Loading positions | Time = 4.7311 ms\n",
+      "-----------------------------------------\n",
+      "size = 5626187\n",
+      "-----------------------------------------\n",
+      "| Loading reference sequence | Time = 458.6 us\n",
+      "-----------------------------------------\n",
+      "-----------------------------------------\n",
+      "| Loading reference accumulative lengths | Time = 31.859 us\n",
+      "-----------------------------------------\n",
+      "[2020-07-10 17:59:46.876] [jointLog] [info] done\n",
+      "[2020-07-10 17:59:46.876] [jointLog] [info] Index contained 5,685 targets\n",
+      "[2020-07-10 17:59:46.877] [jointLog] [info] Number of decoys : 0\n",
+      "\n",
+      "\n",
+      "\n",
+      "\n",
+      "[2020-07-10 17:59:47.844] [jointLog] [info] Automatically detected most likely library type as IU\n",
+      "\u001b[A\r",
+      "\r",
+      "\u001b[32mprocessed\u001b[31m 500,000 \u001b[32mfragments\u001b[0m\n",
+      "hits: 185,982, hits per frag:  0.375185\u001b[A\r",
+      "\r",
+      "\u001b[32mprocessed\u001b[31m 1,000,000 \u001b[32mfragments\u001b[0m\n",
+      "hits: 373,929, hits per frag:  0.378164\u001b[A\r",
+      "\r",
+      "\u001b[32mprocessed\u001b[31m 1,500,000 \u001b[32mfragments\u001b[0m\n",
+      "hits: 560,277, hits per frag:  0.381996\u001b[A\r",
+      "\r",
+      "\u001b[32mprocessed\u001b[31m 2,000,000 \u001b[32mfragments\u001b[0m\n",
+      "hits: 749,888, hits per frag:  0.377336\u001b[A\r",
+      "\r",
+      "\u001b[32mprocessed\u001b[31m 2,500,000 \u001b[32mfragments\u001b[0m\n",
+      "hits: 940,876, hits per frag:  0.379057\u001b[A\r",
+      "\r",
+      "\u001b[32mprocessed\u001b[31m 3,000,000 \u001b[32mfragments\u001b[0m\n",
+      "hits: 1,129,712, hits per frag:  0.38306\n",
+      "\n",
+      "\n",
+      "\n",
+      "\n",
+      "\n",
+      "\n",
+      "\n",
+      "[2020-07-10 17:59:59.702] [jointLog] [info] Computed 7,767 rich equivalence classes for further processing\n",
+      "[2020-07-10 17:59:59.702] [jointLog] [info] Counted 1,149,139 total reads in the equivalence classes \n",
+      "[2020-07-10 17:59:59.704] [jointLog] [i
```

Comment:
```suggestion
    "If we processed the data correctly, we'd expect that genes that are PAO1-specific (ie. those PAO1 genes that do not have a PA14 homolog) typically have very low (0) expression in PA14 samples. We do see this - blue has many genes that have 0 mean gene expression, while the red does not. "
```

Reply:
This is not quite working, but basically just that sentence is a bit hard to follow so I suggested a way to rephrase it a little.

Reply:
Thanks I have gone back to reword this so hopefully its more clear

Reply:
Sorry if my comment seemed rude -- I meant something was going wrong with the suggest interface and I wasn't actually suggesting you change your log output but rather the text!

Reply:
Not at all rude! I knew what you meant no worries